### ISSUE TITLE
Cannot install under node:8

### ISSUE THREAD

**Issue by hakman** [sematext/logagent-js#121]
I am having problems installing under Node 8:
```
Dockerfile 
FROM node:8
RUN \
  npm install -g @sematext/logagent
```
The result is:
```
$ docker build .
Sending build context to Docker daemon  2.048kB
Step 1/2 : FROM node:8
 ---> b87c2ad8344d
Step 2/2 : RUN   npm install -g @sematext/logagent
 ---> Running in c8ae3510c38f
npm WARN deprecated sprintf@0.1.5: The sprintf package is deprecated in favor of sprintf-js.
/usr/local/bin/logagent-setup -> /usr/local/lib/node_modules/@sematext/logagent/bin/logagent-setup.sh
/usr/local/bin/logagent -> /usr/local/lib/node_modules/@sematext/logagent/bin/logagent.js
/usr/local/bin/logagent-windows -> /usr/local/lib/node_modules/@sematext/logagent/bin/logagent-windows.js
/usr/local/bin/logagent-uninstall -> /usr/local/lib/node_modules/@sematext/logagent/bin/logagent-uninstall.sh

> microtime@2.1.7 install /usr/local/lib/node_modules/@sematext/logagent/node_modules/microtime
> prebuild-install || node-gyp rebuild

prebuild-install info begin Prebuild-install version 2.5.0
prebuild-install info looking for local prebuild @ prebuilds/microtime-v2.1.7-node-v57-linux-x64.tar.gz
prebuild-install WARN install EACCES: permission denied, access '/root/.npm'
gyp WARN EACCES user "undefined" does not have permission to access the dev dir "/root/.node-gyp/8.9.4"
gyp WARN EACCES attempting to reinstall using temporary dev dir "/usr/local/lib/node_modules/@sematext/logagent/node_modules/microtime/.node-gyp"
gyp WARN EACCES user "nobody" does not have permission to access the dev dir "/usr/local/lib/node_modules/@sematext/logagent/node_modules/microtime/.node-gyp/8.9.4"
gyp WARN EACCES attempting to reinstall using temporary dev dir "/usr/local/lib/node_modules/@sematext/logagent/node_modules/microtime/.node-gyp"
gyp WARN EACCES user "nobody" does not have permission to access the dev dir "/usr/local/lib/node_modules/@sematext/logagent/node_modules/microtime/.node-gyp/8.9.4"
```

The workaround is to use:
```npm i -g --unsafe-perm @sematext/logagent ```

**Comment by wadey** [comment]
It looks like you are running npm as the root user. This is not supported by prebuild-install, and you need to add the --unsafe-perm flag. Please open the issue with https://github.com/prebuild/prebuild-install .

**Comment by wadey** [comment]
Actually see this long ticket that is discussing the same issue: https://github.com/nodejs/node-gyp/issues/454#issuecomment-315691803

**Comment by megastef** [comment]
@wadey thank you for the pointers. Logagent runs sometimes as service, the above example is from LA Docker image, which worked fine before we got this issue  by a dependency to `graygelf`, which uses microtime. 
Anyhow an endless loop during installation is a serious problem ..., we might remove the dependency to  `graygelf` until we have a better solution. CC: @fbalicchia

**Comment by megastef** [comment]
Removed the optional `graygelf` dependency, released Logagent 2.0.105 and updated documentation for GELF/graylog users to install `graygelf` separately: 
https://sematext.com/docs/logagent/input-plugin-gelf/ 
https://sematext.com/docs/logagent/output-plugin-gelf/
